src/Product.php

```diff
@@ -69,147 +73,157 @@ public function toArray() {
   }
 
   /**
+   * Get the product name.
+   *
    * @return string
+   *   The name.
    */
   public function getName() {
     return $this->name;
   }
 
   /**
+   * Set the product name.
+   *
    * @param string $name
+   *   The name.
    *
-   * @return Product
+   * @return Drupal\commerce_google_tag_manager\Product
+   *   The Product object.
    */
   public function setName($name) {
     $this->name = $name;
     return $this;
   }
 
   /**
+   * Get the unique identifier.
+   *
    * @return string
+   *   The unique identifier.
    */
   public function getId() {
     return $this->id;
   }
 
   /**
+   * Set the unique identifier.
+   *
    * @param string $id
+   *   The identifier.
    *
-   * @return Product
+   * @return Drupal\commerce_google_tag_manager\Product
+   *   The Product object.
    */
   public function setId($id) {
     $this->id = $id;
     return $this;
   }
 
   /**
+   * Get the price.
+   *
    * @return string
+   *   The price.
    */
   public function getPrice() {
     return $this->price;
   }
 
   /**
+   * Set the price.
+   *
    * @param string $price
+   *   The price.
    *
-   * @return Product
+   * @return Drupal\commerce_google_tag_manager\Product
+   *   The Product object.
    */
   public function setPrice($price) {
     $this->price = $price;
     return $this;
   }
 
   /**
-   * @return string
-   */
-  public function getBrand() {
```

Comment:
I was not sure if this was really necessary (never use in the code am I wrong ?)